cypress/integration/aaaaaaaaaaaaaaaaaaaaaaaa/add_a_product_to_a_pricelist_schema_and_create_a_new_PLV.js

```diff
@@ -0,0 +1,184 @@
+/*
```

Comment:
Can we move it from `aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa` ?

Reply:
Oh my, yes, i'll move it tomorrow, sorry. I generally keep it in an "aaaa" named folder because it is the first test to run, so if it were to fail, it fails soon, so i can cancel the whole jenkins task.